### ISSUE TITLE
Does it support HK only?

### ISSUE THREAD

**Issue by oujinlong** [tommyinb/PokemonGoMap#4]
Thank for sharing
Does it support HK only?

**Comment by tommyinb** [comment]
Yes. But you can change the monitoring area by altering 3 lines.
1) DownloadData.cs line 19: change bound in URL to suit your area
2) bin/index.html line 234: change to the center location of your area
3) bin/Debug/Account.xml: create an XML file for Account.cs to let program commit results itself
Then, run the program, a new area is scanned.
